resources/views/layouts/app.blade.php

```diff
@@ -33,7 +36,7 @@
         });
     </script>
 </head>
-<body>
+<body onload="init()">
```

Comment:
Note to @SplitSet3R/front-end  I suggest everyone to put anything that requires onload in init() function in custom.js